data-lineage/src/main/java/com/wipro/ats/bdre/lineage/dataaccess/LineageDao.java

```diff
@@ -110,7 +110,7 @@ public static void closeConnection() throws SQLException {
 					closeResultset(rs);
 					closeConnection();
 				} catch (SQLException ex) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)